src/components/HighlightableMenuItem.tsx

```diff
@@ -0,0 +1,36 @@
+import type {ForwardedRef} from 'react';
+import {forwardRef} from 'react';
```

Comment:
Looks like we missed adding `React` here.
![Screenshot 2024-04-04 at 5 59 35 PM](https://github.com/Expensify/App/assets/3004659/98e4ee40-7de4-49a0-bfb2-e15f664eb24f)

Reply:
It's weird that there is no lint / type error for this. I've added it anyways.